html/css/dark.css

```diff
@@ -1,6 +1,3 @@
-  body {
-    background: #333;
```

Comment:
you sure about this?

Reply:
Yes, body background is set further down.  Just removing the stupid unused one :)